systemverilog-plugin/uhdmcommonfrontend.cc

```diff
@@ -78,8 +78,9 @@ void UhdmCommonFrontend::execute(std::istream *&f, std::string filename, std::ve
     bool dump_vlog2 = false;
     bool no_dump_ptr = false;
     bool dump_rtlil = false;
+    std::vector<std::string> unhandled_args;
 
-    for (size_t i = 1; i < args.size(); i++) {
```

Comment:
What is the reason for changing that? Was the first argument skipped? It is `1` in Verilog frontend, so I assume `args[0]` is the command name?

Reply:
Yes, ``args[0]`` is command name.
Surelog ``parseCommandLine`` expects that first argument is program name. Up to now, we were always passing whole args, so for ``read_systemverilog -debug test.v`` we were passing ``[read_systemverilog, -debug, test.v]`` to Surelog (but we were skipping ``read_systemverilog`` in our internal plugin loop. After this change, we pass ``[read_systemverilog, test.v]`` (as we already processed ``-debug`` flag).
We could add anything to the ``args[0]`` for Surelog, but I think it is better just to pass command name, as we are skipping it in our internal loop.

Reply:
Sure, passing the command in this case seems OK. Just add a comment explaining this, just in case something changes in `parseCommandLine` and we run into weird errors down the way.

Reply:
Ok, true, added.